docker/docker-compose-api-worker.yml

```diff
@@ -2,7 +2,7 @@ version: '2.4'
 name: mojito
 services:
   db:
-    image: mysql:5.7
+    image: mysql:8.0.34
```

Comment:
I feel this change shouldn't in this same PR, but have its own PR instead.

Reply:
Agree, I'll move this to a separate PR.

Reply:
New PR: https://github.com/pinterest/mojito/pull/33